src/components/PageHero.tsx

```diff
@@ -118,11 +44,51 @@ const PageHero: React.FC<IProps> = ({
   const { buttons, title, header, subtitle, image, alt } = content
   return (
     <Content>
```

Comment:
We need to migrate this component as well.

Reply:
Resolved